app/controllers/admin/users_controller.rb

```diff
@@ -31,7 +34,7 @@ def verify
     @user.verified = !@user.verified
     @user.save!
     render json: { success: true }
-  rescue => e
+  rescue StandardError => e
```

Comment:
Isn't this the default?

Reply:
It is. I didn't find a consistent pattern throughout the app, it feels like evenly split between explicitly mentioning `StandardError` or leave it implicit. I tend to prefer the former because I think rescuing errors is a sensitive action, but I don't feel that strongly about it so I can remove it for now.